fraqbot/Local/shakespeare.py

```diff
@@ -1,49 +1,59 @@
-from Legobot.Lego import Lego
 import logging
 import os
 import random
 import sys
 
+from Legobot.Lego import Lego
+
+
 logger = logging.getLogger(__name__)
+LOCAL_DIR = os.path.join(os.path.abspath(os.path.dirname(__file__)))
 
+if LOCAL_DIR not in sys.path:
+    sys.path.append(LOCAL_DIR)
 
-class Shakespeare(Lego):
-    insult_array = []
 
+import helpers as h  # noqa E402
+
+
+class Shakespeare(Lego):
     def __init__(self, baseplate, lock, *args, **kwargs):
         super().__init__(baseplate, lock, acl=kwargs.get('acl'))
+        self.insult_array = []
+        insults = h.load_file(
+            os.path.join(LOCAL_DIR, 'lists', 'quotes.txt'), raw=True)
 
-        LOCAL_DIR = os.path.join(os.path.abspath(os.path.dirname(__file__)))
-        if LOCAL_DIR not in sys.path:
-            sys.path.append(LOCAL_DIR)
-
-        with open(os.path.join(LOCAL_DIR, 'lists/quotes.txt')) as my_file:
-            for line in my_file:
-                Shakespeare.insult_array.append(line)
+        if insults:
```

Comment:
as long as you get back a result, we split on line breaks and store it in the class's insult_array property